tests/test_schema_docs.py

```diff
@@ -0,0 +1,106 @@
+"""Test that docs contain links to the latest schemas."""
+
+import pathlib
+import re
+
+import pytest
+
+repo_path = pathlib.Path(__file__).parent.parent
+docs_path = repo_path / "docs"
+transforms_docs = docs_path / "source" / "jwst" / "transforms" / "index.rst"
+transforms_resources_path = repo_path / "src" / "stdatamodels" / "jwst" / "transforms" / "resources"
+transforms_schemas_path = transforms_resources_path / "schemas" / "stsci.edu" / "jwst_pipeline"
+legacy_transforms_schemas_path = (
```

Comment:
nitpick, but isn't it more normal to have these defined constants in all caps?

Reply:
Good call. I'll update the PR.

Reply:
How does 4e78314f797a1cdc2fb0cf6a1141209a7c37b243 look?